### PR TITLE
MAINT-27977: Remove platform/administrators permission from old created space news folder

### PR DESCRIPTION
**Pull request by Jihed525**
This Backport in needed to apply [patch](https://github.com/exoplatform/news/pull/42) on eXo 5.3.x

### CODE REVIEW

File: upgrade-plugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  http://www.fsf.org. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.exoplatform.addons.upgrade</groupId>
+    <artifactId>upgrade</artifactId>
+    <version>5.3.x-SNAPSHOT</version>
+  </parent>
```
Comment (azayati) [inline upgrade-plugin/pom.xml:20]:
```suggestion
<parent>
    <artifactId>news</artifactId>
    <groupId>org.exoplatform.addons.news</groupId>
    <version>1.1.x-SNAPSHOT</version>
  </parent>
```

File: upgrade-plugin/pom.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  <artifactId>data-upgrade-news</artifactId>
```
Comment (azayati) [inline upgrade-plugin/pom.xml]:
```suggestion
  <artifactId>news-upgrade-plugin</artifactId>
```

File: upgrade-plugin/pom.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  <packaging>jar</packaging>
+  <name>eXo Add-on:: Data Upgrade Add-on - NEWS</name>
```
Comment (azayati) [inline upgrade-plugin/pom.xml]:
```suggestion
  <name>eXo Add-on :: NEWS Add-on Upgrade Plugin</name>
```

**Review by azayati — CHANGES_REQUESTED** [review]
See review

**Review by azayati — CHANGES_REQUESTED** [review]
https://github.com/exoplatform/news/pull/42 this PR should be backported also

File: upgrade-plugin/src/main/resources/conf/portal/configuration.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        </value-param>
+        <value-param>
+          <name>plugin.upgrade.async.execution</name>
+          <description>The plugin will be executed in an asynchronous mode</description>
+          <value>true</value>
+        </value-param>
+        <value-param>
+          <name>plugin.upgrade.target.version</name>
+          <description>Target version of the plugin</description>
+          <value>6.0.0</value>
```
Comment (azayati) [inline upgrade-plugin/src/main/resources/conf/portal/configuration.xml]: Should be 5.3.5

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: upgrade-plugin/src/main/resources/conf/portal/configuration.xml (hunk 4 above)
Comment (Jihed525) [reply upgrade-plugin/src/main/resources/conf/portal/configuration.xml]: done

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**